api/data_refinery_api/settings.py

```diff
@@ -159,6 +161,25 @@
     'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.AcceptHeaderVersioning'
 }
 
+# Setting the RAVEN_CONFIG when RAVEN_DSN isn't set will cause the
+# following warning:
+# /usr/local/lib/python3.6/site-packages/raven/conf/remote.py:91:
+# UserWarning: Transport selection via DSN is deprecated. You should
+# explicitly pass the transport class to Client() instead.
+raven_dsn = get_env_variable('RAVEN_DSN', "not set")
+if raven_dsn != "not set":
+    RAVEN_CONFIG = {
+        'dsn': raven_dsn,
+        # Only send 5% of errors for the API, since we aren't going to
+        # be interested in any single one.
+        'sampleRate': 0.05
```

Comment:
Let's crank this to 15/25 for the first release then lower it, I don't want to miss anything while we're still low traffic.